modules/claims_api/spec/requests/v2/veterans/rswag_power_of_attorney_spec.rb

```diff
@@ -632,37 +639,112 @@
+          schema type: :object, properties: [{ status: { type: :string } }]
+          let(:veteranId) { '1012667145V762142' } # rubocop:disable RSpec/VariableName
+          let(:data) do
+            SwaggerSharedComponents::V2.body_examples[:power_of_attorney_2122][:schema][:example] # rubocop:disable Naming/VariableNumber
+          end
+
+          before do |example|
+            Veteran::Service::Representative.new(representative_id: '67890',
+                                                 poa_codes: [organization_poa_code],
+                                                 first_name: 'Firstname',
+                                                 last_name: 'Lastname',
+                                                 phone: '555-555-5555').save!
+            Veteran::Service::Organization.create(poa: organization_poa_code,
+                                                  name: "#{organization_poa_code} - DISABLED AMERICAN VETERANS")
+            mock_ccg(scopes) do |auth_header|
+              Authorization = auth_header # rubocop:disable Naming/ConstantName
+              submit_request(example.metadata)
+            end
+          end
+
+          after do |example|
+            example.metadata[:response][:content] = {
+              'application/json' => {
+                example: JSON.parse(response.body, symbolize_names: true)
+              }
+            }
+          end
+
+          it 'returns a valid 200 response' do |example|
+            assert_response_matches_metadata(example.metadata)
           end
         end
       end
 
-      describe 'Getting a 401 response' do
-        response '401', 'Unauthorized' do
-          it 'returns a 401 response' do
-            one = 1
-            expect(one).to eq(1)
+      describe 'Getting a 422 response' do
+        response '422', 'Bad Request' do
+          schema type: :object, properties: [{ status: { type: :string } }]
+          let(:veteranId) { '1012667145V762142' } # rubocop:disable RSpec/VariableName
+          let(:data) do
+            temp = SwaggerSharedComponents::V2.body_examples[:power_of_attorney_2122][:schema][:example] # rubocop:disable Naming/VariableNumber
+            temp['data']['attributes']['serviceOrganization']['poaCode'] = nil
+            temp
+          end
+
+          before do |example|
+            mock_ccg(scopes) do |auth_header|
+              Authorization = auth_header # rubocop:disable Naming/ConstantName
+              submit_request(example.metadata)
+            end
+          end
+
+          after do |example|
+            example.metadata[:response][:content] = {
+              'application/json' => {
+                example: JSON.parse(response.body, symbolize_names: true)
+              }
+            }
+          end
+
+          it 'returns a 422 response' do |example|
+            assert_response_matches_metadata(example.metadata)
+          end
+        end
+      end
+
+      describe 'Getting a 404 response' do
```

Comment:
Should this be a 404?

Reply:
I would have to look at what exactly is happening once we get the /validate docs merged in. it does look like this file has been changed a good bit, so we may have this resolved?